Directory.Build.props

```
@@ -2,6 +2,16 @@
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
   <Import Condition="'$(EnvironmentBuildPropsImported)' != 'True'" Project="$(MSBuildThisFileDirectory)eng\Environment.Build.props" />
 
+  <PropertyGroup>
+    <!-- 
+      Keep backward compatibility with XamlC, XamlCompilationAttribute, and xaml-comp processing instruction
+      When we're ready to turn this off, we can remove all code depending on this condition, and drop this property group
+      -->
+
+    <_MauiXamlSourceGenBackCompat>true</_MauiXamlSourceGenBackCompat>
```

For testing, should you allow someone to turn this off?
```suggestion
    <_MauiXamlSourceGenBackCompat Condition="'$(_MauiXamlSourceGenBackCompat)' == ''">true</_MauiXamlSourceGenBackCompat>
```